src/tests/unit.spec.ts

```diff
@@ -0,0 +1,171 @@
+import axios from 'axios';
+import { AxiosHarTracker } from '../axios-har-tracker'
+import MockAdapter from 'axios-mock-adapter';
+
+describe('Check axios-har-tracker', () => {
```

Comment:
This unit test is testing exactly the same thing as e2e tests above, it just operates on mocks instead of calling real test endpoints so in fact it's not a unit test. The idea of unit tests is to test each unit (method) in AxiosTracker class separately, for example: `generateNewEntry`, `pushNewEntryResponse`, `getCookies`, etc. For now I think it's enough to have good e2e tests calling test endpoints and the unit tests coverage can be done in separate PR.

Reply:
so I will create 2nd PR for unit tests, after we will merge this one